addons/website/static/src/components/dialog/add_page_dialog.xml

```diff
@@ -43,6 +43,11 @@
 <t t-name="website.AddPageTemplatePreviews">
     <div class="container-fluid py-3 py-xl-4">
         <div class="row g-0">
+            <div t-if="this.props.isCustom and !this.props.templates.length" class="col-12 alert alert-info d-flex align-items-center">
+                <i class="fa fa-info-circle fa-lg fa-stack d-flex align-items-center justify-content-center me-3 p-2 rounded-1"/>
```

Comment:
I would rather do a design a bit like the snippet alert.

```suggestion
            <div t-if="this.props.isCustom and !this.props.templates.length" class="col-12 alert alert-info d-flex">
                <i class="fa fa-2x fa-info-circle me-3"/>
```

before

![image](https://github.com/odoo/odoo/assets/52911687/7b966fbf-e63a-4141-99ef-6cb90c2fc299)

after

![image](https://github.com/odoo/odoo/assets/52911687/4f2506a5-072b-4615-942a-fd5e6071ba27)